geometry_msgs/Pose pose
bool gripper_open
float32 battery_level
sensor_msgs/Range[] ultrasonic_sensors
